DataFormats/Common/src/RefHolder.cc

```diff
@@ -0,0 +1,8 @@
+#include <DataFormats/Common/interface/RefHolderBase.h>
```

Comment:
```suggestion
#include "DataFormats/Common/interface/RefHolderBase.h"
```
We tend to use the quotation marks for `#include`s inside CMSSW.

Reply:
Quotes force the compiler to make an extra lookup in the current folder where we already know it will find nothing.

Reply:
Point taken, but our policy is to use quotes for headers inside CMSSW. Please change.

Reply:
Thanks!